src/index.js

```diff
@@ -126,6 +126,10 @@ export function withField (el) {
   const wrappedComponent = markRaw({
     name: 'withFieldWrapper',
     props: {
+      label: {
+        type: String,
+        default: undefined
```

Comment:
suggestion NB: If were going with type String, this should probably be `''`. Should be enough to satisfy `!` conditions, pending what @logaretm thinks

Reply:
No problem here, using either is safe.